### ISSUE TITLE
Undefined method `buffer_chunk_limit'

### ISSUE THREAD

**Issue by Raznak** [fluent/fluentd#1025]
Hello,

I'm using Fluentd on FreeBSD and I have an issue when I configure Fluentd to Mongo:

2016-06-03 16:27:54 +0200 [error]: unexpected error error="undefined method `buffer_chunk_limit' for #<Fluent::Plugin::FileBuffer:00000803295698>"

There is the configuration file

```
<source>
  @type tail
  format /^"(?<app_name>[^"]*)" (?<src_ip>[^ ]*) [^ ]* (?<user>[^ ]*) \[(?<time>[^\]]*)\] "(?<http_method>\S+)(?: +(?<requested_uri>[^ ]*) +(?<incoming_protocol>[\S]*))?" (?<http_code>[^ ]*) (?<size>[^ ]*)(?: "(?<referer>[^\"]*)" "(?<user_agent>[^\"]*)")? (?<bytes_received>[0-9]+) (?<bytes_sent>[0-9]+) (?<time_elapsed>[0-9]+)$/
  time_format %d/%b/%Y:%H:%M:%S %z
  types code:integer,bytes_sent:integer,bytes_received:integer,size:integer,time_elapsed:integer
  path /var/log/apache2/access.log
  pos_file /var/log/fluentd/574ffb8ab9a6d00a29de9904.pos
  tag apache.access.574ffb8ab9a6d00a29de9904.raw
</source>

<match apache.access.574ffb8ab9a6d00a29de9904.*>

  # Internal MongoDB repository configuration
  @type mongo_replset
  database logs
  collection access
  ssl true
  ssl_cert /var/db/mongodb/mongod.pem
  ssl_key /var/db/mongodb/mongod.pem
  nodes xxx:9091
  
  flush_interval 10s
  include_time_key true
  buffer_type memory
  buffer_chunk_limit 128m
  buffer_queue_limit 32
  retry_wait 1s

</match>
```

If I remove the lines for the buffer, the result is the same....

Thanks

**Comment by Raznak** [comment]
When I comment the line 106 in the out_mongo.rb
 -> `@buffer.buffer_chunk_limit = available_buffer_chunk_limit`

fluentd start without issues....

**Comment by repeatedly** [comment]
I forgot to release new version for v0.14.0.
I just released v0.7.13. Could you test this version?

**Comment by Raznak** [comment]
Yes it works !
Thanx !
